Editor/Resources/EditorWindow/GameLiftPlugin.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using AmazonGameLift.Editor;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Editor.Resources.EditorWindow
+{
+    public class GameLiftPlugin : UnityEditor.EditorWindow
```

Comment:
This can just be 

```
public class GameLiftPlugin: EditorWindow
```

since you are declaring the import `using UnityEditor` ?

Reply:
No, since the type `EditorWindow` clashes with the current namespace